src/render/TextRenderer.js

```diff
@@ -91,53 +109,49 @@ define([
         };
 
         /**
-         * Creates a texture for a specified text string, a specified font and an optional outline.
-         * @param {DrawContext} dc The current draw context.
+         * Creates a texture for a specified text string.
          * @param {String} text The text string.
-         * @param {Font} font The font to use.
-         * @param {Boolean} outline Indicates whether the text is drawn with a thin black outline.
          * @returns {Texture} A texture for the specified text string and font.
          */
-        TextRenderer.prototype.createTexture = function (dc, text, font, outline) {
-            var gl = dc.currentGlContext,
-                ctx2D = this.ctx2D,
+        TextRenderer.prototype.createTexture = function (text) {
```

Comment:
This should be refactored into two methods, TextRender.renderText(text) and TextRenderer.drawText(text) to conform to the Android model.

Reply:
Right! I forgot about splitting up the function. Will do it ASAP.

Reply:
@markpet49 Reconsidering splitting up the function, maybe it would be better to keep it a single one in WebWW.

`drawText()` in WWA is only used to assist `renderText()` in creating a bitmap, which is a concept that we don't apply in WebWW. Plus it's a `protected` function, and nothing else in the project calls it. As outlined in #176, the refactor focuses in public methods.

That being said, I should have renamed `createTexture()` in WebWW as `renderText()` in order to bring it up in line with WWA's version. They have the same signature and both are used by analogous classes in both projects.

What do you think?